imagedephi/redact/tiff.py

```diff
@@ -115,8 +135,8 @@ def report_missing_rules(self) -> None:
 
     def report_plan(self) -> None:
         print("Tiff Metadata Redaction Plan\n")
-        for rule in self.redaction_steps.values():
-            print(rule.get_description())
+        for tag_value, rule in self.redaction_steps.items():
+            print(f"Tiff Tag {tag_value} - {rule.key_name}: {rule.action}")
         self.report_missing_rules()
```

Comment:
Because the file format is now tied to the `RedactionPlan` alone, and no longer the individual rules, I've moved the logic of printing out rules to the `RedactionPlan` classes, but if another home makes sense (like the `TiffRules` and `SvsRules` classes), let me know.